app/router.js

```diff
@@ -8,7 +8,7 @@ var Router = Ember.Router.extend({
 Router.map(function() {
   this.route('foo', function() {
     this.route('bar', function() {
-      this.route('baz');
+      this.route('baz', function() {});
```

Comment:
This use of the `function` as an argument to `this.route` enables the ability to use `loading` and `index` templates.